src/app/give-help/transport-form/transport-form.component.html

```diff
@@ -125,6 +125,8 @@
               (input)="onPhoneNumberChange($event)"
               [(ngModel)]="phoneNumber"
               appPhoneValidate
+              minlength="7"
+              maxlength="15"
```

Comment:
Wg ticketa tu ma być max 22, a o min nie było mowy. Była zmiana decyzji?

Reply:
Hej, tak poruszałem ten temat z Dominiką

Reply:
OK, dzięki za info. To może warto to dopisać do Jiry, by nie uciekło. Poza tym OK.